AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/Status.java

```diff
@@ -27,11 +25,11 @@
 
     public enum Type {
```

Comment:
Of course. So that means that I'm wrong with my assuption that e.g. status `Skull` is deactivated? It just looed like that since te code is greyed out. Then again, I could be wrong here.